docs/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
I assume this is just here so that the snapshot has a license attached too? Could it just be at the very top level of dafny-lang.github.io instead?

Reply:
It is there also. I can delete it from the current doc version if you like, though I'd prefer to keep the current docs and the snapshots as identical as possible.